apps/bfd-server/bfd-server-war/src/test/resources/endpoint-responses/v2/claimSearch.json

```diff
@@ -1,14 +1,14 @@
 {
   "resourceType": "Bundle",
-  "id": "1856fb88-119e-46b6-a775-ab78ae0f61e9",
+  "id": "d24acebc-2239-41a7-91ee-4f1e38202453",
   "meta": {
-    "lastUpdated": "1970-01-01T00:00:01.000+00:00"
+    "lastUpdated": "2021-12-06T18:43:02.797+00:00"
   },
   "type": "searchset",
   "link": [
     {
       "relation": "self",
-      "url": "https://localhost:6500/v2/fhir/Claim?mbi=a7f8e93f09&service-date=gt1970-07-18&service-date=lt1970-07-30"
+      "url": "https://localhost:9327/v2/fhir/Claim?mbi=a7f8e93f09&service-date=gt1970-07-18&service-date=lt1970-07-30"
```

Comment:
why the change in the port number?

Reply:
I'm copying the output from the test result itself and then just verifying that only the things that were supposed to change did, just easier than copying over the value of one changed field at a time.  The port is random, so it changes sometimes, but the test ignores that value in it's comparison, so it doesn't affect the test result.